FreeRTOS_UDP_IP.c

```diff
@@ -573,6 +582,27 @@ static BaseType_t prvHandleUDPPacketWithoutSocket( NetworkBufferDescriptor_t * p
         else
     #endif /* ipconfigUSE_LLMNR */
 
+    #if ( ipconfigUSE_MDNS == 1 )
+        /* A MDNS request, check for the destination port. */
+        if( ( usPort == FreeRTOS_ntohs( ipMDNS_PORT ) ) ||
+            ( pxProtocolHeaders->xUDPHeader.usSourcePort == FreeRTOS_ntohs( ipMDNS_PORT ) ) )
```

Comment:
It seems a bit odd to bypass using a normal socket when handling DNS/MDNS/LLMNR/NBNS response packets.